recipe/patches/0003-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch

```diff
@@ -1,28 +0,0 @@
-From d11ba2d6dbd21931bcc9233a0b553a56781917cf Mon Sep 17 00:00:00 2001
-From: Ray Donnelly <mingw.android@gmail.com>
-Date: Wed, 16 Aug 2017 11:59:00 +0100
-Subject: [PATCH 03/30] Win32: distutils: Add support to cygwinccompiler for
- VS>=2015
-
-Really this is for mingw-w64 though
----
- Lib/distutils/cygwinccompiler.py | 3 +++
- 1 file changed, 3 insertions(+)
-
-diff --git a/setuptools/_distutils/cygwinccompiler.py b/setuptools/_distutils/cygwinccompiler.py
-index 66c12dd358..e6c790118b 100644
---- a/setuptools/_distutils/cygwinccompiler.py
-+++ b/setuptools/_distutils/cygwinccompiler.py
-@@ -80,6 +80,9 @@ def get_msvcr():
-         elif msc_ver == '1600':
-             # VS2010 / MSVC 10.0
-             return ['msvcr100']
-+        elif int(msc_ver) >= 1900:
```

Comment:
The upstream interval is >=1900 and <2000. seems to be OK more "correct" than this patch.